api/registrations/serializers.py

```diff
@@ -11,17 +12,71 @@ class RegistrationSerializer(NodeSerializer):
     retracted = ser.BooleanField(source='is_retracted', read_only=True,
         help_text='Whether this registration has been retracted.')
     date_registered = ser.DateTimeField(source='registered_date', read_only=True, help_text='Date time of registration.')
+    retraction_justification = ser.CharField(source='retraction.justification', read_only=True)
+    pending_retraction = HideIfRetraction(ser.BooleanField(source='is_pending_retraction', read_only=True,
+        help_text='Is this registration pending retraction?'))
+    pending_registration_approval = HideIfRetraction(ser.BooleanField(source='sanction.pending_approval', read_only=True,
+        help_text='Does this registration have a sanction pending approval?'))
+    pending_embargo = HideIfRetraction(ser.BooleanField(read_only=True, source='is_pending_embargo',
+        help_text='Is this registration pending embargo?'))
+    registered_meta = HideIfRetraction(ser.SerializerMethodField(
```

Comment:
`registered_meta` and `registration_supplement` will need to be updated for compatibility with the schema changes in the draft reg feature.